core/services/privacy/policy/PrivacyPolicyManager.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace EventEspresso\core\services\privacy\policy;
+
+use EventEspresso\core\exceptions\InvalidClassException;
+use EventEspresso\core\exceptions\InvalidDataTypeException;
+use EventEspresso\core\exceptions\InvalidEntityException;
+use EventEspresso\core\exceptions\InvalidFilePathException;
+use EventEspresso\core\exceptions\InvalidIdentifierException;
+use EventEspresso\core\exceptions\InvalidInterfaceException;
+use EventEspresso\core\services\collections\CollectionDetails;
+use EventEspresso\core\services\collections\CollectionInterface;
+use EventEspresso\core\services\collections\CollectionLoader;
+use WP_Post;
+
+/**
+ * Class PrivacyPolicyManager
+ * Manages setting up the hooks to add the EE core and add-ons' privacy policies
+ *
+ * @package        Event Espresso
+ * @author         Mike Nelson
+ * @since          $VID:$
+ */
+class PrivacyPolicyManager
+{
+
+    public function __construct()
+    {
+        if (function_exists('wp_add_privacy_policy_content')) {
```

Comment:
Maybe we should move this function check into EE_System and add it to the conditional that decides whether to even load this class? May as well avoid the extra loading and memory usage if it's going to be unnecessary.

Reply:
sure